content/post/106-ipfs-meetup-aug2020.md

```diff
@@ -0,0 +1,51 @@
+The IPFS community got together last week for the August virtual meetup. This event continues a series that we started with the [IPFS 0.5 Launch Meetup](https://www.youtube.com/watch?v=G8FvB_0HlCE), and continued with the IPFS Pinning Summit and a IPFS June Meetup to kick off the summer.
+
+But one sense, this particular meetup *is* a change: These virtual meetups are now replacing the IPFS Weekly community meeting. The IPFS Weekly has been held on most Mondays at 9am Pacific time for several years - host to loads of speakers from within PL and from the broader community, sharing their ideas, thoughts, products and experiments with IPFS, libp2p and various related technologies. Thanks to all the people who joined in real time and who commented on the recordings. So now, instead of a shorter weekly session with fewer people, we are building on the success of the larger events by doing them more regularly. You can look forward to more people joining, in-depth talks and a wide variety of speakers, projects and demos from around the dweb ecosystem.
```

Comment:
```suggestion
But one sense, this particular meetup *is* a change: These virtual meetups are now replacing the IPFS Weekly community meeting. The IPFS Weekly has been held on most Mondays at 9am Pacific time for several years - host to loads of speakers from throughout the broader community, sharing their ideas, thoughts, products and experiments with IPFS, libp2p and various related technologies. Thanks to all the people who joined in real time and who commented on the recordings. So now, instead of a shorter weekly session with fewer people, we are building on the success of the larger events by doing them more regularly. You can look forward to more people joining, in-depth talks and a wide variety of speakers, projects and demos from around the dweb ecosystem.
```